docs/for_developers.md

```diff
@@ -236,6 +236,25 @@ in submodules can lead to duplicated `<version>` declarations with different ver
 All version numbers should be easily audited, and so should be in a single file (i.e., the
 *root* `pom.xml`).
 
+
+### THIRD-PARTY LIBRARIES
```

Comment:
One of the libraries used by the regex distance code has BSD license, would that be compatible?

Reply:
As far as I know, only GPL is a problem. other OS licenses are usually fine